docs/reference/pattern_detection_and_enhancement.md

```diff
@@ -1,10 +1,10 @@
 ## Pattern detection and metadata enhancement
 
-The PatternDetector is responsible for identifying specific patterns within the input, such as using regular expressions to detect ISSN, ISBN, DOI, and PMID. Other techniques than regular expressions may also occur here, such as doing phrase matching to identify known scientific journals, or fingerprint matching to identify librarian curated responses.
+A PatternDetector is responsible for identifying specific patterns within the input, such as using regular expressions to detect ISSN, ISBN, DOI, and PMID (implemented in our StandardIdentifiers Class). Other techniques than regular expressions may also occur as PatternDetectors, such as doing phrase matching to identify known scientific journals, or fingerprint matching to identify librarian curated responses.
```

Comment:
This is non-blocking: it may be useful to write this as `Pattern Detector` rather than `PatternDetector`, so it's clear that we're talking about a concept and not a model/module.